config/locales/en/views/laa_references.yml

```diff
@@ -2,7 +2,7 @@
 en:
   laa_reference:
     link:
-      failure: A link to the court data source could not be created. %{error_messages}
+      failure: A Court Data Source link could not be established, due to an invalid MAAT Reference Number. Please check the MAAT Reference Number.
```

Comment:
Commenting fully as a grammar pedant, the first sentence would read better without the comma

Reply:
That's strange, thought I made this change already - will re-review

Reply:
Yeah this has been resolved - code is from an older commit: 
![image](https://user-images.githubusercontent.com/26544538/154515652-478b4e1f-09f7-4d61-ae8a-1ecaea889a7e.png)